assets/constants.ts

```diff
@@ -451,13 +451,17 @@ export = {
   "ALLOWED_QUESTION_INTERACTION_CATEGORIES": [{
     "name": "General",
     "interaction_ids": [
+      "ImageClickInput",
+      "ItemSelectionInput",
       "MultipleChoiceInput",
       "TextInput"
     ]
   }, {
     "name": "Math",
     "interaction_ids": [
-      "NumericInput"
+      "FractionInput",
+      "NumericInput",
+      "NumberWithUnits"
```

Comment:
If you are ordering alphabetically... This should be above NumericInput...

Reply:
Done